bridge/bridgeapp/views.py

```diff
@@ -27,9 +28,13 @@ def get(self, request):
 
 class BridgeCategoryView(View):
     """Get Thread table from database and display threads associated with Category selected on Home Page"""
-    def get(self, request, category_id, category_slug):  #category_slug needed for UI
+
+    def get(self, request, category_id, category_slug):  # category_slug needed for UI
+        # assigns list of category object id's to category variable
```

Comment:
Inaccurate statement. Here you take the _**single**_ category from the url `category_id` and assign it to a variable.

No list is invovled.